contentcuration/contentcuration/frontend/channelList/views/ChannelSet/ChannelSetModal.vue

```diff
@@ -199,7 +202,7 @@
     computed: {
       ...mapGetters('channelSet', ['getChannelSet']),
       isNew() {
-        return Boolean(this.channelSet[NEW_OBJECT]);
+        return !this.channelSetId || this.$route.path === '/collections/new';
```

Comment:
Are there scenarios where `channelSetId` is empty(not set) and we are not creating a new collection?